packages/mdc-drawer/component.ts

```diff
@@ -34,13 +34,19 @@ import {MDCDrawerFocusTrapFactory} from './util';
 
 const {cssClasses, strings} = MDCDismissibleDrawerFoundation;
 
+/**
+ * @events MDCDrawer:event %-% Emits when the navigation drawer has closed. %-% {}
```

Comment:
is this `@event` (Singular)? is this annotation valid for class name? I do not see any examples.

Reply:
`@event` for some reason does not work with documentalist. I believe it is a bug - I'll open one against documentalist.

Reply:
https://github.com/palantir/documentalist/issues/93